README.md

```diff
@@ -4,7 +4,7 @@
 
 [![R-CMD-check](https://github.com/Olink-Proteomics/OlinkRPackage/actions/workflows/R-CMD-check.yaml/badge.svg)](https://github.com/Olink-Proteomics/OlinkRPackage/actions/workflows/R-CMD-check.yaml)
 [![lint](https://github.com/Olink-Proteomics/OlinkRPackage/actions/workflows/lint.yaml/badge.svg)](https://github.com/Olink-Proteomics/OlinkRPackage/actions/workflows/lint.yaml)
-[![codecov.io](https://codecov.io/gh/r-lib/lintr/branch/main/graphs/badge.svg)](https://github.com/Olink-Proteomics/OlinkRPackage/actions/workflows/test-coverage.yaml)
+[![Coverage](https://img.shields.io/badge/Coverage-100%25-green)](https://img.shields.io/badge/Coverage-100%25-green)
```

Comment:
Coverage is being shown as 100% even though the coverage is not 100%.  Running locally with 1 added file to decrease the test levels: 
![image](https://github.com/user-attachments/assets/af169594-477b-490c-a24c-67842bab987d)
![image](https://github.com/user-attachments/assets/c0e45195-432c-4db8-9e81-b8d27041b43f)
However on CI it is showing as 100%
![image](https://github.com/user-attachments/assets/a311ae61-247f-4f1d-8f34-1f69cf053cfe)
![image](https://github.com/user-attachments/assets/c7fb820a-ee7e-4c9c-b012-ae374db22fbb)

Reply:
See PR #539 for the exact code and examples

Reply:
I tested branch test_ci_cov used in #539 for its coverage and it is:
<img width="232" alt="Screenshot 2025-03-14 153138" src="https://github.com/user-attachments/assets/a7f14124-38e9-4889-8e80-bc3978028c6a" />
Rounding this number up gives 100% again.

I did something more drastic to lower the test coverage. I removed all tests for read_npx_zip and this this the calculated coverage locally:
<img width="233" alt="Screenshot 2025-03-14 153457" src="https://github.com/user-attachments/assets/9d76ed94-55dd-4e3e-b644-16a0ce750287" />

And after I pushed on test_ci_cov used in #539, the badge was updated
<img width="530" alt="Screenshot 2025-03-14 154129" src="https://github.com/user-attachments/assets/a0666df4-7c25-4041-bc72-259499b06c00" />